vj4/handler/base.py

```diff
@@ -268,6 +268,10 @@ def json(self, obj):
     self.response.write(data)
 
   @property
+  def page_category(self):
```

Comment:
page_category is fixed for a handler class, consider using a constant instead of a property.

Reply:
It varies for contest handlers because both contest and homework use the contest handler.